WASM.md

```diff
@@ -1,23 +1,64 @@
-Proof of Concept WebAssembly Support.
+WebAssembly Extension Support
 
-# WebAssembly VM
+# Dependencies
 
-The WebAssembly VM is https://github.com/WAVM/WAVM. This VM was chosen for the
-proof of concept because:
+## apt-get install libglib2.0-dev (on Debian/Ubuntu).
```

Comment:
Nit: split name (glibc headers) from the instructions, like for all other dependencies.

Reply:
done.